### PR TITLE
fix(Sticky): `scrollContext` is used from actual props, rename doc examples

### PR DESCRIPTION
**Pull request by layershifter**

### CODE REVIEW

File: docs/app/Examples/modules/Sticky/Types/StickyExampleAdjacentContext.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Grid, Header, Image, Rail, Segment, Sticky } from 'semantic-ui-react'
 
 const Placeholder = () => <Image src='/assets/images/wireframe/paragraph.png' />
 
-export default class StickyAdjacentContextExample extends Component {
+export default class StickyExampleAdjacentContext extends Component {
```
Comment (layershifter) [inline docs/app/Examples/modules/Sticky/Types/StickyExampleAdjacentContext.js:7]: All examples should name by template `${Component}Example${Name}`

File: docs/app/Examples/modules/Sticky/Types/index.js [hunk 2]
```diff
@@ -7,13 +7,13 @@ const StickyTypesExamples = () => (
...
-      description='Sticky content attaches itself to the viewport when it is passed'
-      examplePath='modules/Sticky/Types/StickyAdjacentContextExample'
+      description='Sticky content attaches itself to the viewport when it is passed.'
+      examplePath='modules/Sticky/Types/StickyExampleAdjacentContext'
     />
     <ComponentExample
       title='Pushing'
-      description='It will have the viewport "push" the sticky content depending on the scroll direction'
-      examplePath='modules/Sticky/Types/StickyPushingExample'
+      description='It will have the viewport "push" the sticky content depending on the scroll direction.'
```
Comment (layershifter) [inline docs/app/Examples/modules/Sticky/Types/index.js:15]: Dots :smile:

File: docs/app/Examples/modules/Sticky/Variations/StickyExampleActive.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { Checkbox, Grid, Header, Image, Rail, Segment, Sticky } from 'semantic-u
 
 const Placeholder = () => <Image src='/assets/images/wireframe/paragraph.png' />
 
-export default class StickyPushingExample extends Component {
+export default class StickyExampleActive extends Component {
```
Comment (layershifter) [inline docs/app/Examples/modules/Sticky/Variations/StickyExampleActive.js:7]: Typo

File: src/modules/Sticky/Sticky.js [hunk 4]
```diff
@@ -98,42 +98,43 @@ export default class Sticky extends Component {
...
+    if (active) this.removeListeners()
   }
 
   // ----------------------------------------
   // Events
   // ----------------------------------------
 
-  addListener = () => {
-    const { scrollContext } = this.props
+  addListeners = (props) => {
```
Comment (layershifter) [inline src/modules/Sticky/Sticky.js:130]: As method adds multiple listeners now, I corrected its name

File: src/modules/Sticky/Sticky.js [hunk 5]
```diff
@@ -98,42 +98,43 @@ export default class Sticky extends Component {
...
-  componentWillReceiveProps({ active: next }) {
+  componentWillReceiveProps(nextProps) {
     const { active: current } = this.props
+    const { active: next } = nextProps
 
     if (current === next) return
     if (next) {
       this.handleUpdate()
-      this.addListener()
+      this.addListeners(nextProps)
```
Comment (layershifter) [inline src/modules/Sticky/Sticky.js:112]: `addListeners` uses `scrollContext` prop, it should have actual value

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.74.2`

**Pull request merged**